plugins/services/src/js/constants/ServiceStatusLabels.ts

```diff
@@ -1,6 +1,17 @@
 import { i18nMark } from "@lingui/react";
 
-const ServiceStatusLabels = {
+interface ServiceStatusLabelsInterface {
+  RUNNING: string;
+  DEPLOYING: string;
+  STOPPED: string;
+  NA: string;
+  DELAYED: string;
+  WAITING: string;
+  DELETING: string;
+  RECOVERING: string;
```

Comment:
Seems like there's some duplication between this and `ServiceStatusInterface`. I wonder if we could define the keys separately and then use them like `{ [key: keyof <some reusable type for keys>]: string }`. Or otherwise put the `ServiceStatusInterface` and this interface in the same file (seems like if one has to change the other would also have to change).

Reply:
I agree that we could refactor that part of the code, but I don't think that's worth doing in this PR.